topics/metabolomics/tutorials/msPurity_training/tutorial.md

```diff
@@ -0,0 +1,1006 @@
+>   - **MSP schema to use for files** : choose the schema you want for your `msp` file. You have 2 options : MAssBank or MoNa. For the tutorial `keep the default parameter`.
+>   - **Filter peaks that have been flagged in prior processing steps** : during the filtering step some peaks can have been flagged because they didn't respect threshold or parameters we completed. Here you have to choose if you want to keep or not this flagged peaks. For the tutorial `keep it to "yes"` to filter these flagged peaks.
+>   - **Always include the following adduct descriptions in the MSP file** : if you want to check some adducts, select them in the list. For our tutorial `keep it empty` to have the default adduct [M+H]+.
+>   - **Create MSP spectra for each adduct?** : here you choose if you want one file per adduct. You should `keep it to "yes"` to have different file for different adducts.
+>
+{: .hands_on}
+
+So when you look at your output, you should have something like this : 
+![MSP file example](../../images/tutorial_msms_msp_example.png "MSP file example (MassBank format)")
+
+- *RECORD_TITLE* : the title given for this peak. **msPurity** take informations about its mass (MZ), retention time (RT), group id (grpid), file, adduct and compile all of them to build the name. 
+{: .text-justify}
+- *MS$FOCUSED_ION* : the exact mass of the precursor.
+{: .text-justify}
+- *AC$CHROMATOGRAPHY* : the exact retention time of the precursor.
+{: .text-justify}
+- *XCMS groupid (grpid)* : the group id in which the precursor has been classed.
+{: .text-justify}
+- *COMMENT* : some words about how we obtained this peak and which tool we used to.
+{: .text-justify}
+- *PK$NUM_PEAK* : number of peaks in the MS/MS spectra.
+{: .text-justify}
+- *PK$PEAK* : the column names of the list of peaks containing in the MS/MS spectra, follow by all the peaks in lines below.  
+{: .text-justify}
+
+We can now use this file to make our annotation with different software like *MetFrag*, *Sirius CSI-Finger-ID*, etc.... This can be done on Galaxy (and describe later in this tutorial) or you can also take informations directly in your `msp` file and work peak by peak in [this website](https://msbi.ipb-halle.de/MetFragBeta/ "MetFrag web version") for example.
+{: .text-justify}
+
+## 6 - (Optionnal) Create a database
+
+This function can be used to create an SQLite database for the LC-MS/MS example experiment. It can be done just after the averaging to be sure that we will create the database of our spectra already sorted. 
+The SQLite schema of the spectral database can be detailed here : 
+
+![Full schema of the database it will create ](../../images/tutorial_msms_DB_createDatabase.png "Full schema of the database created by this tool")
+
+Add more explanations about the schema  !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
I have to add more explanations about the schema of database...
How is it build ? Why building it like this ? Some things like that